script/client

```diff
@@ -173,6 +173,8 @@ GetOptions(
     'asset-size-limit:i', 'with-thumbnails', 'accept=s',    'yaml-output',
 ) or usage(1);
 
+warn 'openqa-client is deprecated and is planned to be removed in the near future. Please use openqa-cli instead';
```

Comment:
```suggestion
warn "WARNING: openqa-client is deprecated and is planned to be removed in the near future. Please use openqa-cli instead\n";
```
to suppress the line number at the end (which makes it not really look like a user-facing output) and to raise the attention level.

Maybe it makes also sense to use `print STDERR ` instead.

Reply:
Ok, changed the text and the newline, that makes sense. `warn` and `print STDERR` is likely ending up in just the same but I prefer the less characters of `warn` :)